testexec/defaults.go

```diff
@@ -46,3 +50,28 @@ func defaultAssertFn(t *testing.T, actual, expect string) {
 		t.Errorf("expected: %q; actual: %q", expect, actual)
 	}
 }
+
+// Hash Table of all the "special" nodes used by testmark.
+var DefaultLeafNodes = map[string]struct{}{
+	"exitcode": {},
+	"stderr":   {},
+	"stdout":   {},
+	"output":   {},
+	"input":    {},
+	"sequence": {},
+	"script":   {},
+	"fs":       {},
+}
+
+func RecursionFn_Then(t *testing.T, dir testmark.DirEnt) error {
```

Comment:
Exporting this (and the table of leaf nodes) seems unnecessary.  I cannot conceive of a use case where someone would want to chain together something that needs to call this.

Reply:
The leaf nodes don't need to be. But extending the base behavior by wrapping the function seems like a normal pattern that I would use.